### DISCUSSION TITLE
Two-factor authentication method Problem.

### DISCUSSION THREAD

**Discussion by waqarakrambhutta** [community/community#112783] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Two-factor auth methods:
[I'm ](https://github.com/waqarakrambhutta) from Pakistan. I'm trying to add SMS/Text message authentication method but in Country or region code section there's no Country **Pakistan** or code **+92** showing. Why is that?? 
[Link](https://github.com/settings/security?type=sms#two-factor-summary)

**Comment by DNin01** [comment]
GitHub doesn't support SMS 2FA in countries that have unreliable text message delivery as it leads to people becoming locked out of their accounts when GitHub can't reliably deliver one-time codes via text. Learn more in the [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported).

You might not be able to use SMS for 2FA, but you can always use an authenticator app instead.
